shopelectro/views/catalog.py

```diff
@@ -242,7 +253,7 @@ def template_context(page, tag_titles, tags):
             'total_products': total_products,
             'products_count': (page_number - 1) * products_on_page + products.count(),
             'paginated_page': paginated_page,
-            'sorting_options': config.category_sorting(),
+            'sorting_options': settings.CATEGORY_SORTING_OPTIONS.values(),
```

Comment:
It may be better to use it as a method of `CategorySortingOption`

Reply:
@artemiy312 , y, it'll be good to hide every sorting options processing logic in `SortingOption` abstraction.
But `SortingOption` can't do this himself in this case.

We should create class `SortingOptions` for it. And right now it'll have only one purpose: call dict `values` method inside. And it'll have only one use case here.
So, here i use settings var directly